sidebars.js

```diff
@@ -13,21 +13,46 @@
 
 /** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
 const sidebars = {
-  // By default, Docusaurus generates a sidebar from the docs folder structure
-  docSidebar: [{type: 'autogenerated', dirName: '.'}],
-
-  // But you can create a sidebar manually
-  /*
-  tutorialSidebar: [
-    'intro',
-    'hello',
+  docSidebar: [
+    {type: 'autogenerated', dirName: '.'},
     {
-      type: 'category',
-      label: 'Tutorial',
-      items: ['tutorial-basics/create-a-document'],
-    },
+      'Resources': [
+        {
+          'Client Library References': [
```

Comment:
We don't currently have a link for it, but [the https://github.com/divviup/divviup-api/tree/main/client] will have rustdocs, and the cli will have documentation somewhere (maybe a README?)

Reply:
I can add a temporary link for the DU client. For CLI docs perhaps our best bet is just to use this repository?

Reply:
My only worry there is that it's hard enough to keep changes within a repo synced and even harder to keep documentation synced with an external repo

Reply:
That's fair. I'll put a link to https://github.com/divviup/divviup-api/tree/main/cli, which will show a README once we have one.